pkg/clients/edgeconnect/iface.go

```diff
@@ -6,7 +6,7 @@ type Client interface {
 	GetEdgeConnect(edgeConnectId string) (GetResponse, error)
 
 	// CreateEdgeConnect creates edge connect
-	CreateEdgeConnect(name string, hostPatterns []string, hostMappings []HostMapping, oauthClientId string) (CreateResponse, error)
+	CreateEdgeConnect(name string, hostPatterns []string, hostMappings []HostMapping, oauthClientId string, managedByOperator ...bool) (CreateResponse, error)
```

Comment:
I really dislike this 😬 (reading this in a few months will 100% confuse people)

it's a golang limitation that we can't have default values in methods, but I don't think this is the way around it. (feels super hacky)

I would just update the interface to have `managedByOperator bool` and set it explicitly where used.
(or not do it in general)
or have the `Request` be the input, so you can change it more freely

Reply:
`managedByOperator` argument is mandatory. It's meaning is clear. Just in case the production version (`!e2e`) of `newRequest` function doesn't allow to make a mistake.

Reply:
@albertogdd could you please remind me: do we have ticket for refactor this ? or should we create new one?